src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    <p className="Person__age">
+      {person.age && `I am ${person.age}`}
+    </p>
+    <p className="Person__partner">
+      {!person.isMarried
+        ? 'I am not married'
+        : `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
```suggestion
        : `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
```

винеси в змінну, тут подвійна вкладеність тернарних операторів